velox/type/tests/TimestampTest.cpp

```diff
@@ -91,5 +91,19 @@ TEST(TimestampTest, arithmeticOverflow) {
       ts2.toNanos(), "integer overflow: -9223372036854776 * 1000000000");
 }
 
+TEST(TimestampTest, toAppend) {
+  std::string tsString1;
+  toAppend(Timestamp(0, 0), &tsString1);
```

Comment:
perhaps, `EXPECT_EQ(..., folly::to<std::string>(Timestamp(0, 0))`

Reply:
Isn't the only point of adding toAppend function is to enable folly::to? Otherwise, we won't have this function.

Reply:
Thats correct for now, but doesn't limit to folly::to
I can add tests covering folly::to in addition to directly testing toAppend - Does that work?